tools/test-proxy/Azure.Sdk.Tools.TestProxy/README.md

```diff
@@ -115,29 +115,38 @@ The azure-sdk team regularly update the image associated with the `latest` tag.
 To ensure that your local copy is up to date, run:
 
 ```powershell
-> docker pull azsdkengsys.azurecr.io/engsys/testproxy-lin:latest
+> docker pull azsdkengsys.azurecr.io/engsys/test-proxy:latest
 ```
 
 ## Command line arguments
 
 This is the help information for test-proxy. It uses the nuget package [`CommandLineParser`](https://www.nuget.org/packages/CommandLineParser) to parse arguments.
 
+The test-proxy executable fulfills one of two primary purposes:
+
+1. The test-proxy server (the only option up to this point)
+2. `asset-sync` push/restore/reset.
+
+This is surfaced by only showing options for the default commands. Each individual command has its own argument set that can be detailed by invoking `test-proxy <command> --help`.
+
+
 ```text
-Azure.Sdk.Tools.TestProxy
-  test-proxy
+/>test-proxy --help
+Azure.Sdk.Tools.TestProxy 1.0.0-dev.20220926.1
+c Microsoft Corporation. All rights reserved.
+
+  start      (Default Verb) Start the TestProxy.
+
+  push       Push the assets, referenced by assets.json, into git.
 
-Usage:
-  Azure.Sdk.Tools.TestProxy [options] [<args>...]
+  reset      Reset the assets, referenced by assets.json, from git to their original files referenced by the tag. Will prompt
+             if there's pending changes.
```

Comment:
I deserve to burn for this level of grammar nitpicking:

```suggestion
             if there are pending changes.
```

Reply:
Technically correct is the best type of correct.